src/winetricks

```diff
@@ -11971,6 +11971,67 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    w_package_unsupported_win64
+
+    if [ ! -x "$(which ntlm_auth 2>/dev/null)" ]; then
+        w_die "winbind (part of samba) is required for the installation"
+    fi
+
+    #link from https://www.askvg.com/direct-download-link-microsoft-office-2013-professional-plus-free-trial/
+    w_download http://care.dlservice.microsoft.com/dl/download/2/9/C/29CC45EF-4CDA-4710-9FB3-1489786570A1/OfficeProfessionalPlus_x86_en-us.img 236f8faae3f979ec72592a63784bba2f0d614916350c44631221b88ae9dae206 "OFFICE15.iso"
+
+    w_set_winver win7
+
+    w_call corefonts
+    w_call riched20
+
+
+    if w_workaround_wine_bug 43581 "wine has problems parsing some regex strings during installation."; then
+        w_call msxml6
+    fi
+
+    if w_workaround_wine_bug 38648 "DirectX < 11 has problems with black window after installation"; then
```

Comment:
Is this needed on Wine 3.0+?  If not, a version number should be added, e.g. `,2.22` to run for all Wine versions less than 3.0. See https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L2542 and https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L2563

Reply:
I confirmed Wine 2.22 needs this and 3.0 doesn't need on Intel HD Graphics 510.

```sh
    if w_workaround_wine_bug 38648 "Wine < 3.0 has problems with black window after installation" ,3.0; then
```

If adding `,2.22`, it's not applied when using Wine 2.22.

Here is a test verb (`wwbtest.verb`):

```sh
w_metadata wwbtest settings \
    title="w_workaround_wine_bug test"

load_wwbtest()
{
    "$WINE" --version

    if w_workaround_wine_bug 38648 "test" ,3.0; then
        echo "Wine < 3.0"
    else
        echo "Wine 3.0 or newer"
    fi

    if w_workaround_wine_bug 38648 "test2" ,2.22; then
        echo "Wine < 2.22"
    else
        echo "Wine 2.22 or newer"
    fi
}
```

Results:

```
[Wine 2.11]
wine-2.11                                                                      
------------------------------------------------------                         
Working around wine bug 38648 -- test
------------------------------------------------------
Wine < 3.0                                                                     
------------------------------------------------------                         
Working around wine bug 38648 -- test2
------------------------------------------------------
Wine < 2.22

[Wine 2.22]
wine-2.22                                                                      
------------------------------------------------------                         
Working around wine bug 38648 -- test
------------------------------------------------------
Wine < 3.0                                                                     
Current Wine does not have Wine bug 38648, so not applying workaround          
Wine 2.22 or newer

[Wine 3.0]
wine-3.0                                                                       
Current Wine does not have Wine bug 38648, so not applying workaround          
Wine 3.0 or newer                                                              
Current Wine does not have Wine bug 38648, so not applying workaround          
Wine 2.22 or newer
```

Reply:
https://github.com/Winetricks/winetricks/pull/939/commits/b2369077b8c95c54cb0abebaba27035880da712f

We could do dimilar for msxml as I did not require it when installing through wine-3.0 and wine-2.1